internal/server/cache/cache.go

```diff
@@ -49,8 +49,10 @@ type Cache struct {
 	blocklistSvgs map[string]struct{}
 	// SVG search index
 	svgSearchIndex *resource.SearchIndex
-	// Custom provenance from SQL storage
-	customProvenances map[string]*pb.Facet
+	// Provenance from SQL storage
+	sqlProvenances map[string]*pb.Facet
+	// SQL database entity, variable existence pairs
+	sqlEntityVariable map[string]map[string]struct{}
```

Comment:
This article talks about using struct keys in such situations. Consider using it if it makes things easier. https://go.dev/blog/maps#key-types

Reply:
This is a very good ideal. Updated